app/views/policy_makings/edit.html.erb

```diff
@@ -0,0 +1,12 @@
+<h1>PolicyMakings#edit</h1>
+
+<p> Policy Making id : <%= @policy_making.id %> </p>
```

Comment:
I don't think the id will be very relevant to the user, do you?

Reply:
Nope, this view was just here for debugging, I didn't do the front end yet